### PR TITLE
fix claim reward issue

### PR DESCRIPTION
**Pull request by XiaoYhun**

### CODE REVIEW

File: src/hooks/useClaimReward.ts [hunk 1]
```diff
@@ -93,14 +93,15 @@ export default function useClaimReward() {
     if (rewardContract && chainId && account && library && data && userRewards[phaseId]) {
       setAttemptingTxn(true)
       //execute isValidClaim method to pre-check
+      const userReward = userRewards[phaseId]
```
Comment (viet-nv) [inline src/hooks/useClaimReward.ts:96]: can you define type for `userRewards`?

File: src/hooks/useClaimReward.ts [hunk 2]
```diff
@@ -93,14 +93,15 @@ export default function useClaimReward() {
...
+          userReward.reward.index,
           account,
-          data.tokens,
-          userRewards[phaseId].amounts,
-          userRewards[phaseId].proof,
+          userReward.tokens,
+          userReward.reward.amounts,
+          userReward.reward.proof,
         )
         .then((res: any) => {
```
Comment (nguyenhuudungz) [inline src/hooks/useClaimReward.ts:106]: Can you change `any`?

File: src/hooks/useClaimReward.ts [hunk 3]
```diff
@@ -112,18 +113,19 @@ export default function useClaimReward() {
...
```
Comment (nguyenhuudungz) [inline src/hooks/useClaimReward.ts:113]: Can you change `any`?

Comment (viet-nv) [inline src/hooks/useClaimReward.ts:113]: why do we need 2 `.then` here?
btw, can you add type for response from contract?

**Review by viet-nv — CHANGES_REQUESTED** [review]

**Review by viet-nv — APPROVED** [review]

**Pull request merged**